examples/next/pages/ui/components/authenticator/useAuthenticator/index.page.tsx

```diff
@@ -13,7 +13,7 @@ export default function App() {
 
   return (
     <Authenticator>
-      {() => <button onClick={navigateHome}>Navigate to Home</button>}
+      <button onClick={navigateHome}>Navigate to Home</button>
```

Comment:
We use render props in a few of our authenticator examples. Should we update those as part of this PR?

Reply:
I think we should leave it as is.

Reply:
Yeah, render props are still our official happy case. We can reevaluate this at 3.0, but I'm good to leave them as-is now.